Source/WebKit/UIProcess/Cocoa/FullscreenClient.h

```diff
@@ -63,21 +63,21 @@ class FullscreenClient final : public API::FullscreenClient {
 
     struct {
 #if PLATFORM(MAC)
-        bool webViewWillEnterFullscreen : 1;
-        bool webViewDidEnterFullscreen : 1;
-        bool webViewWillExitFullscreen : 1;
-        bool webViewDidExitFullscreen : 1;
+        bool webViewWillEnterFullscreen : 1 = false;
```

Comment:
I think we generally prefer `{ false };` instead of `= false;`, from a quick scan of our sources.